packages/create-flex-plugin/templates/ts/src/states/CustomTaskListState.ts

```diff
@@ -0,0 +1,33 @@
+import {Action} from './index';
```

Comment:
`import { Action } from './index';`

It would be great to add Prettier to automatically format the documents

Reply:
The problem is, these are all template files; so they are invalid files until they are rendered :(

Reply:
Sadly makes sense :)